src/coverlet.console/Program.cs

```diff
@@ -15,6 +15,42 @@ namespace Coverlet.Console
 {
     class Program
     {
+        /// <summary>
+        /// Exit Codes returned from Coverlet console process.
+        /// </summary>
+        enum CommandExitCodes
+        {
+            /// <summary>
+            /// Indicates successful run of dotnet test without any test failure and coverage percentage above threshold if provided.
+            /// </summary>
+            Success = 0,
+
+            /// <summary>
+            /// Indicates test failure by dotnet test.
+            /// </summary>
+            TestFailed = 1,
+
+            /// <summary>
+            /// Indicates coverage percentage is below given threshold for one or more threshold type.
+            /// </summary>
+            CoverageBelowThreshold = 2,
+
+            /// <summary>
+            /// Indicates test failure by dotnet test and coverage percentage is below given threshold for one or more threshold type.
+            /// </summary>
+            TestFailedAndCoverageBelowThreshold = TestFailed + CoverageBelowThreshold,
```

Comment:
I would remove this...it's not used.

Reply:
Its not directly used but code value of 3 represent this. I thought by adding this in enum, we are kind of reserving and making meaning of exit code value 3 clearer for any reference.

Reply:
With `Flags` this "meaning" is implicit.